src/main/treeDataProviders/utils/dependencyUtils.ts

```diff
@@ -106,12 +105,11 @@ export class DependencyUtils {
             descriptorsParsed.add(child.generalInfo.path);
         }
         this.reportNotFoundDescriptors(descriptorsPaths, descriptorsParsed, scanManager.logManager);
+        if (!contextualScan || bundlesWithIssues.length == 0) {
```

Comment:
Isn't bundlesWithIssues populated asynchronously? If so, bundlesWithIssues will be 0 when its shouldn't